src/clj/game/cards/ice.clj

```diff
@@ -575,7 +575,6 @@
                                                 (str/join ", " (map :title (take 3 (:deck runner))))
                                                 " from the Runner's Stack"))
                                (wait-for (mill state :corp :runner 3)
-                                         (system-msg state side (str "trashes Aimor"))
```

Comment:
Reason for this? Seems relevant for tracking what's happening.

Reply:
You're right, I was misled by the fact Aimor was the only trap ice logging its own trashing.

I found out there might be room for improvement in this area, so I'd like to work on it a bit more